AdyenTests/Adyen Tests/Components/Apple Pay/ApplePayComponentTest.swift

```diff
@@ -69,6 +69,23 @@ class ApplePayComponentTest: XCTestCase {
         waitForExpectations(timeout: 2)
     }
     
+    func testApplePayViewControllerIsResetAfterAuthorization() {
```

Comment:
As I mentioned yesterday - this test case is excessive:

we already have tests for `paymentAuthorizationViewController(didAuthorizePayment:)` and `stopLoading`.

Reply:
its necessary to test that the `viewController` is nilled after calling stopLoading, right?

Reply:
it is already [tested here](https://github.com/Adyen/adyen-ios/blob/d478372a95e4d798120e42e3385f3e20350f68cc/AdyenTests/Adyen%20Tests/Components/Apple%20Pay/ApplePayComponentTest.swift#L37-L43)

Reply:
ok, I will remove it, but in another PR, is that okay?